dependency-check-suppressions.xml

```diff
@@ -2,7 +2,7 @@
 <suppressions
 	xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
-	<suppress until="2021-10-25">
+	<suppress until="2021-11-25">
```

Comment:
Ensure this file is reset once this bracnh is brought up to match `master` branch.